xml/ha_clvm.xml

```diff
@@ -348,6 +345,139 @@ cLVM) for more information and details to integrate here - really helpful-->
   /dev/sdb vgtest b</screen>
     </step>
    </procedure>
+   <para>To test your clustered mirror device, proceed as follows:</para>
```

Comment:
I cannot add comments there, so put them here:
1. <screen>mirror_legs_require_separate_pvs = 1</screen> is wrong, which should be
mirror_logs_require_separate_pvs = 1